src/output.cpp

```diff
@@ -52,12 +52,9 @@ extern bool test_mode;
 
 void delwin_functor::operator()( WINDOW *w ) const
 {
-    if( w == nullptr ) {
-        return;
+    if( w != nullptr ) {
+        delwin( w );
     }
-    werase( w );
-    wrefresh( w );
-    delwin( w );
```

Comment:
Other code may rely on this. The `WINDOW_PTR` is already used at several places which might require those `werase`, `wrefresh` calls.